zipline/utils/run_algo.py

```diff
@@ -168,9 +169,12 @@ def _run(handle_data,
         def choose_loader(column):
             if column in USEquityPricing.columns:
                 return pipeline_loader
-            raise ValueError(
-                "No PipelineLoader registered for column %s." % column
-            )
+            elif column in loaders:
```

Comment:
I think this will crash in the current implementation if the user doesn't supply any custom loaders (because `loaders` will be None, so `column in loaders` will barf).